templates/workspaces/base/porter.yaml

```diff
@@ -82,7 +82,7 @@ uninstall:
   - terraform:
       description: "Tear down workspace"
       # https://github.com/microsoft/AzureTRE/issues/1526
-      logLevel: Debug
+      # logLevel: Debug
```

Comment:
nit: is this intentionally left commented out?

Reply:
Yes, I will enable it in my other pr that deals with resource processor logging